Services/ImageSaverService/Package.swift

```diff
@@ -0,0 +1,22 @@
+// swift-tools-version: 6.0
+
+import PackageDescription
+
+let package = Package(
+    name: "ImageSaverService",
```

Comment:
```suggestion
    name: "ImageService",
```

Let's keep it generic to include more features in future? like picking an image for your wallet

Reply:
Or `ImageGalleryService` something more intutive

Reply:
resolved